crowbar_framework/app/controllers/repositories_controller.rb

```diff
@@ -49,12 +49,14 @@ def sync
   # required parameters: platform, repo
   def activate
     return render_not_found if params[:platform].nil? || params[:repo].nil?
+    ret, message = ProvisionerService.new(logger).enable_repository(params[:platform], params[:repo])
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `message`. Use `_` or `_message` as a variable name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)